docs/chainlink-keepers/best-practices.md

```diff
@@ -8,81 +8,86 @@ whatsnext:
 
-- **Manage unbounded upkeeps** - limit the problem set of your on-chain execution by creating a range bound for your Upkeep to check and perform. This allows you to keep within predefined gas limits, resulting in a predictable upper bound gas cost on your transactions. Break apart your problem into multiple Upkeep registrations to limit the scope of work.
+- **Manage unbounded upkeeps**: Limit the problem set of your on-chain execution by creating a range bound for your Upkeep to check and perform. This allows you to keep within predefined gas limits, which creates a predictable upper bound gas cost on your transactions. Break apart your problem into multiple Upkeep registrations to limit the scope of work.
 
-  For example, create an Upkeep for each subset of addresses to be serviced - from 0 to 49, 50 to 99, 100 to 149.
+  **Example**: You could create an Upkeep for each subset of addresses that you want to service. The ranges could be 0 to 49, 50 to 99, and 100 to 149.
 
-- **Manage code paths** - pass in data to your `checkUpkeep` to make your contract logic go down different code paths. This can be used in creative ways based on your use case needs.
+- **Manage code paths**: pass in data to your `checkUpkeep` to make your contract logic go down different code paths. This can be used in creative ways based on your use case needs.
 
-  For example, you could support multiple types of Upkeep within a single contract, and pass a function selector via your `checkData`.
+  **Example**: You could support multiple types of Upkeep within a single contract, and pass a function selector through the `checkData` function.
 
 ## Testing
 
 As with all smart contract testing, it is important to test the boundaries of your smart contract in order to ensure it operates as intended. Similarly, it is important to make sure your Keeper-compatible contract operates within the parameters of the `KeeperRegistry`.
 
-Your mission critical contracts should have all code paths tested and you should stress-test your contract to confirm the performance and correct operation of your use case under load or adversarial conditions. The Chainlink Keeper Network will continue to operate under stress, but so should your contract. [Reach out](https://forms.gle/WadxnzzjHPtta5Zd9) to us if you need any help, have questions, or feedback for improvement.
+Test all of your mission-critical contracts, and stress-test the contract to confirm the performance and correct operation of your use case under load and adversarial conditions. The Chainlink Keeper Network will continue to operate under stress, but so should your contract. [Reach out](https://forms.gle/WadxnzzjHPtta5Zd9) to us if you need help, have questions, or feedback for improvement.
 
 ## Gas Limits
```

Comment:
Let's move the "Gas Limits" section above "Testing".

Reply:
Updated in latest push.